package.json

```diff
@@ -48,6 +52,19 @@
       "import/no-extraneous-dependencies": 0
     }
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npx lint-staged"
+    }
+  },
+  "lint-staged": {
+    "**/*.{js,vue}": [
+      "eslint --fix",
+      "vue-cli-service lint",
+      "git add"
+    ],
+    "./**": ["git add"]
```

Comment:
What does this line do?

Reply:
I though the root level files weren't getting committed for cases when linter found something to fix.
Just tested and was able to do it now, so removed this line altogether.